packages/mdc-select/README.md

```diff
@@ -80,7 +80,7 @@ Then with JS
 ```js
 import {MDCSelect} from '@material/select';
```

Comment:
should we also then remove this?

Reply:
Yes, but this readme doesn't ever reference the Importing the JS component section, so I'll add that under this code block.